app/controllers/course/condition/assessments_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Course::Condition::AssessmentsController < Course::ConditionsController
+  load_resource :assessment_condition, class: Course::Condition::Assessment.name, parent: false
+  before_action :set_course, only: [:new, :create]
+  authorize_resource :assessment_condition, class: Course::Condition::Assessment.name
+
+  def new
+  end
+
+  def create
+    @assessment_condition.conditional = @conditional
+    @assessment_condition.course = current_course
+
+    if @assessment_condition.save
+      redirect_to return_to_path, success: t('course.condition.assessments.create.success')
```

Comment:
It depends on the route, yes. This controller is inherited? Then you need to use the fully expended translation.